sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Setting `ServiceBusMessage.partition_key` to a value different than `session_id` on the message instance now raises `ValueError`.
 * `ServiceBusSender` and `ServiceBusReceiver` are no longer reusable and will raise `ValueError` when trying to operate on a closed handler.
 * `send_messages`, `schedule_messages`, `cancel_scheduled_messages` and `receive_deferred_messages` now performs a no-op rather than raising a `ValueError` if provided an empty list of messages or an empty batch.
+* `amqp_annotated_message` has been renamed to `raw_amqp_message` to normalize with other SDKs.
```

Comment:
super nit:
```suggestion
* `ServiceBusMessage.amqp_annotated_message` has been renamed to `ServiceBusMessage.raw_amqp_message` to normalize with other SDKs.
```

Reply:
(Since this approved gonna slip this in with our doc changes tomorrow, per our OOB discussion)